assets/sass/v2/_device-challenge.scss

```diff
@@ -0,0 +1,32 @@
+.siw-main-view {
+  .device-challenge-poll {
+    .o-form-fieldset-container {
+      width: 88%;
+      text-align: center;
+      margin: 0 auto;
+
+      .spinner {
+        height: 38px;
+        margin-top: 40px;
+        background: url('../img/ui/indicators/loader@1x.gif') no-repeat center;
+      }
+
+      .link {
+        color: #007dc1;
+      }
+    }
+  }
+
+  .siw-main-footer {
+    .auth-footer {
+      .js-sign-in-options {
```

Comment:
Otherwise it is only shown as grey text. <img width="116" alt="Screen Shot 2020-01-06 at 12 40 01 PM" src="https://user-images.githubusercontent.com/5066836/71847411-ac6d9a80-3081-11ea-9161-b183184d6b5f.png">

Reply:
That's the style across the board. I don't have strong opinion which style to use as long as UX decided but am looking consisency across all pages.

Reply:
@nbories-okta ^

Reply:
@haishengwu-okta the 'sign up' link in the identify footer also have custom css to make it blue. 
![Screen Shot 2020-01-06 at 3 34 36 PM](https://user-images.githubusercontent.com/5066836/71856855-3164ae00-309a-11ea-9735-6962abd44cce.png)

Reply:
yes, that's really a one-off ! 
but we have other `go back` link at other pages. this makes thing more inconsistency.

Reply:
I think it should probably be ok to change the color for `sign up` button link also to `$primary-color`.